plugins/repository-azure/src/test/java/org/opensearch/repositories/azure/AzureBlobContainerRetriesTests.java

```diff
@@ -232,7 +232,7 @@ public void testReadBlobWithRetries() throws Exception {
                         exchange.getResponseHeaders().add("Content-Length", String.valueOf(length));
                         exchange.getResponseHeaders().add("x-ms-blob-type", "blockblob");
                         exchange.getResponseHeaders()
-                            .add("Content-Range", String.format("bytes %d-%d/%d", rangeStart, bytes.length, bytes.length));
+                            .add("Content-Range", String.format("bytes " + rangeStart + "-" + bytes.length + "/" + bytes.length));
```

Comment:
Is the `String.format()` doing anything now?

Reply:
Oh ... definitely not, I forgot to remove it, thank!